docusaurus/docs/reactnative/guides/performance_boost.mdx

```diff
@@ -0,0 +1,91 @@
+---
+id: performance-boost
+sidebar_position: 5
+title: Performance Boost
+---
+
+import channelSettings from '../assets/guides/performance-boost/channel_settings.jpeg'
+
+React Native Chat SDK provides feature rich components which caters to all the types of chat messaging applications.
+Although some of the features or components may come with performance cost, which you will want to avoid for the cases of high
+concurrency applications such as livestreaming. In this guide we have noted down some tips for improving the performance of your
+chat application if you are expecting heavy traffic on your chat.
+
+## Channel Settings
+
+Within each channel type, some settings are available that apply to all channels of that type.
+Among these settings are the ability to enable/disable each event type.
+When disabled, events of that type (for channels of this type) will not be passed through to a client's open WebSocket connection.
+It's also important to note that increasing the events enabled on a channel type also increases the load on clients in those channels.
+
+- For Livestream type channels, we recommend disabling Connect, Read Events, and Typing Events. These will cause performance issues and don't generally add to the user experience in these use cases.
+- Also consider using [Slow Mode](https://getstream.io/chat/docs/slow_mode/?language=js#channel-slow-mode) for Livestream events.
```

Comment:
```suggestion
Within each channel type, some settings are available that apply to all channels of that type.
Among these settings is the ability to enable or disable each event type.
While disabled, an event of a selected type will not be passed through to a client's open WebSocket connection in the channel type it has been set.
It's also important to note that increasing the number of enabled events for a channel type also increases the load on clients in those channels.

- For the Livestream channel type, we recommend disabling Connect, Read Events, and Typing Events. These will cause performance issues and don't generally add to the user experience in these use cases.
- Also consider using [Slow Mode](https://getstream.io/chat/docs/slow_mode/?language=js#channel-slow-mode) for Livestream events.
```

> don't generally add to the user experience in these use cases.  

Just a personal preference aside: I would have preferred to emphasize that this is in our experience of something along those lines, as some may find it a bit irking to be set on something being true without qualifiers for that mentioned. But in no way important to me, just figured I'd mention